internal/backend/runtime/omni/controllers/omni/cluster_machine_config_status_test.go

```diff
@@ -530,24 +531,23 @@ func (suite *ClusterMachineConfigStatusSuite) TestSchematicChanges() {
 		suite.Require().NoError(err)
 	}
 
+	expectedFactoryImage := "factory.talos.dev/installer/bbbb:v1.3.0"
```

Comment:
what's `bbbb`, or maybe call unexpected image?

Reply:
`bbbb` is just the schematic ID we use for testing. It is used at the previous step. We can use any string here, we just test with it.

Here we set it as the schematic ID: https://github.com/siderolabs/omni/blob/main/internal/backend/runtime/omni/controllers/omni/cluster_machine_config_status_test.go#L524

And here we assert that it is in the install URL: https://github.com/siderolabs/omni/blob/main/internal/backend/runtime/omni/controllers/omni/cluster_machine_config_status_test.go#L539

The fix I did is not related to it. My fix is: "if we still receive an Talos upgrade request with the previous install image after updating the resources to trigger a new one, do not fail, as it can happen".